plugins/user-authenticators/ldap/test/resources/cosmic.cloud.ldif

```diff
@@ -16,13 +16,13 @@
 # under the License.
```

Comment:
The intention was to have it being used by the LDAP unit tests, but it is not the case it.

![image](https://cloud.githubusercontent.com/assets/5129209/12575960/dfb2d716-c410-11e5-91ed-f6d5571957ea.png)